### DISCUSSION TITLE
Unknown props warnings

### DISCUSSION THREAD

**Discussion by alkanna** [roxiness/routify#361] [Q&A] [answered]
Hey, I'm getting these two messagesin the console, only on my base index page (not on a child component) :
```
<Pages> was created with unknown prop 'scoped'
<Pages> was created with unknown prop 'scopedSync'
```

Is there something obvious I forgot here ?

Thanks

**Comment by ghostdevv** [comment]
This isn't your fault, most of those issues should be solved but some aren't. Try updating to the latest version and see if that fixes it

**Reply by alkanna** [reply]
Alright got it, thanks :)

**Comment by jakobrosenberg** [comment, marked as answer]
As far as I know, Svelte doesn't have any way of disabling those warnings. They are however harmless.

**Reply by ghostdevv** [reply]
It is possible:
```html
<script>
    export let scoped; // Svelte assumes this is a required prop
</script>
```
but we can do
```html
<script>
    export let scoped = {}; // Svelte will now not assume this is a required prop because you are assigning it a default value (ergo no more console errors)
</script>
```

**Reply by jakobrosenberg** [reply]
I'm not sure we're able to do this on behalf of the pages and layouts. We might however be able to check if the component is expecting the prop before we send it.
